tests/speculos/requirements.txt

```diff
@@ -0,0 +1,5 @@
+speculos
+pytest>=6.1.1,<7.0.0
+ledgercomm>=1.1.0,<1.2.0
+ecdsa>=0.16.1,<0.17.0
```

Comment:
Why this requirement on ecdsa? Is is really necessary to forbid 0.17.0?

Reply:
This is a copy-pasta of [this file](https://github.com/LedgerHQ/app-boilerplate/blob/feature/more_ci_and_tests_example/tests/ledgercomm/requirements.txt) (with `speculos` on top).
At this point I don't know why this dependencies have such limitations.